rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/services/ServiceConfig.java

```diff
@@ -66,4 +95,23 @@ ClouddriverService clouddriverService(
         .build()
         .create(ClouddriverService.class);
   }
+
+  /*As part of retrofit2 changes, creating clouddriverservice with retrofit2 API changes */
+  @Bean
+  ClouddriverRetrofit2Service clouddriverRetrofit2Service(OkHttpClient okHttpClient) {
+    ObjectMapper objectMapper =
```

Comment:
Please move this to a separate bean and share it between the retrofit and retrofit2 code.